### PR TITLE
Include sem 0.7.4 features

### PR DESCRIPTION
**Pull request by mactsouk**

### CODE REVIEW

File: sem-reference_5b6968642c7d3a03f89d6bff.md [hunk 1]
```diff
@@ -602,3 +783,4 @@ same output:
 * [Changing Organizations](https://docs.semaphoreci.com/article/29-changing-organizations)
 * [Pipeline YAML reference](https://docs.semaphoreci.com/article/50-pipeline-yaml)
 * [Dashboard YAML reference](https://docs.semaphoreci.com/article/60-dashboards-yaml-reference)
+* [Job YAML reference]()
```
Comment (markoa) [inline sem-reference_5b6968642c7d3a03f89d6bff.md]: this needs something :)

File: sem-reference_5b6968642c7d3a03f89d6bff.md [hunk 2]
```diff
@@ -171,19 +174,35 @@ follows:
 
 Should you wish to learn more about creating new resources, you can visit
 the [Secrets YAML reference](https://docs.semaphoreci.com/article/51-secrets-yaml-reference),
-the [Dashboard YAML reference]()
+the [Dashboard YAML reference](https://docs.semaphoreci.com/article/60-dashboards-yaml-reference),
+the [Jobs YAML reference]()
```
Comment (markoa) [inline sem-reference_5b6968642c7d3a03f89d6bff.md]: empty link

File: sem-reference_5b6968642c7d3a03f89d6bff.md (hunk 1 above)
Comment (mactsouk) [reply sem-reference_5b6968642c7d3a03f89d6bff.md]: @markoa I know: https://github.com/semaphoreci/docs/pull/49 
The `Job YAML reference` page is not published yet.

File: sem-reference_5b6968642c7d3a03f89d6bff.md (hunk 2 above)
Comment (mactsouk) [reply sem-reference_5b6968642c7d3a03f89d6bff.md]: @markoa The same as before: https://github.com/semaphoreci/docs/pull/49
I think that Darko and I will take care of it tomorrow.

File: sem-reference_5b6968642c7d3a03f89d6bff.md [hunk 3]
```diff
@@ -159,8 +161,9 @@ work with resources.
 
 The `sem create` command is used for creating new resources and can be followed
 by the `-f` flag, which should be followed by a valid path to a proper YAML
-file. Currently there exist three types of YAML configuration files that can be
-handled by `sem create`: secrets, dashboards and projects configuration files.
+file. Currently there exist four types of YAML configuration files that can be
```
[the remark this answers is not in the document]
Comment (mactsouk) [reply sem-reference_5b6968642c7d3a03f89d6bff.md]: @markoa I think that `resource files` might be a more appropriate term but `configuration files` was totally wrong. Good point - thanks!

**Pull request merged**